### PR TITLE
[h2olog] define how generated_raw_tracer.cc is generated in regen.mk

### PR DESCRIPTION
**Pull request by gfx**
To follow the code generation manner in h2o.

* [x] checked if it works on macOS
* [x] checked if it works on Ubuntu

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -937,7 +937,7 @@ IF (WITH_H2OLOG)
 
     ADD_CUSTOM_COMMAND(
         OUTPUT  "${CMAKE_CURRENT_SOURCE_DIR}/src/h2olog/generated_raw_tracer.cc"
-        COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/src/h2olog/misc/gen_raw_tracer.py" "${CMAKE_CURRENT_SOURCE_DIR}" "${CMAKE_CURRENT_SOURCE_DIR}/src/h2olog/generated_raw_tracer.cc"
+        COMMAND make -f misc/regen.mk src/h2olog/generated_raw_tracer.cc
         DEPENDS ${H2OLOG_GEN_DEPS} "${CMAKE_CURRENT_SOURCE_DIR}/src/h2olog/misc/gen_raw_tracer.py"
```
Comment (kazuho) [inline CMakeLists.txt:941]: I think we can move the dependency definition to regen.mk and always invoke `make` here?

File: misc/regen.mk [hunk 2]
```diff
@@ -27,6 +27,9 @@ lib/handler/file/templates.c.h: misc/picotemplate-conf.pl lib/handler/file/_temp
 	misc/picotemplate/picotemplate.pl --conf misc/picotemplate-conf.pl lib/handler/file/_templates.c.h || exit 1
 	clang-format -i $@
 
+src/h2olog/generated_raw_tracer.cc: FORCE
```
Comment (kazuho) [inline misc/regen.mk:30]: Re FORCE, please see my other review comment.

**Review by kazuho — COMMENTED** [review]
Thank you for the PR. Looking good, just one suggestion.

File: CMakeLists.txt (hunk 1 above)
Comment (gfx) [reply CMakeLists.txt]: Sure! Done e33a7c9b5

File: misc/regen.mk [hunk 3]
```diff
@@ -27,6 +27,10 @@ lib/handler/file/templates.c.h: misc/picotemplate-conf.pl lib/handler/file/_temp
 	misc/picotemplate/picotemplate.pl --conf misc/picotemplate-conf.pl lib/handler/file/_templates.c.h || exit 1
 	clang-format -i $@
 
+
```
Comment (kazuho) [inline misc/regen.mk:30]:
```suggestion
```

**Review by kazuho — APPROVED** [review]
Thank you for the changes. LGTM other than the suggestion to remove extra blank line that has been added.

**Pull request merged**